perfkitbenchmarker/cloud_tpu.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
This file needs a GetResourceMetadata method, which should be called by publisher.py.

Reply:
Additionally, the child classes can implement this method too, to add cloud-specific metadata.

Reply:
Thanks